src/test/java/net/minestom/server/inventory/click/type/InventoryCopyCursorTest.java

```diff
@@ -0,0 +1,50 @@
+package net.minestom.server.inventory.click.type;
+
+import it.unimi.dsi.fastutil.ints.IntList;
+import net.minestom.server.MinecraftServer;
+import net.minestom.server.inventory.click.ClickInfo;
+import net.minestom.server.item.ItemStack;
+import net.minestom.server.item.Material;
+import org.junit.jupiter.api.Test;
+
+import static net.minestom.server.inventory.click.ClickUtils.assertClick;
+
+public class InventoryCopyCursorTest {
+
+    static {
+        MinecraftServer.init();
```

Comment:
Why is this being initialized? This is not good, you should be using EnvTest most likely

Reply:
previous tests from early 2022 written by TheMode used this so I assumed it was fine - this is 'Required to prevent initialization error during event call'. Should I still move it over to EnvTest?